windows/CodePush.Net46/FileUtils.cs

```diff
@@ -0,0 +1,62 @@
+﻿using PCLStorage;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace CodePush.ReactNative
+{
+    internal class FileUtils
+    {
+        internal async static Task MergeFoldersAsync(IFolder source, IFolder target)
+        {
+            foreach (IFile sourceFile in await source.GetFilesAsync().ConfigureAwait(false))
+            {
+                await CopyFileAsync(sourceFile.Path, Path.Combine(target.Path, sourceFile.Name)).ConfigureAwait(false);
+            }
+
+            foreach (IFolder sourceDirectory in await source.GetFoldersAsync().ConfigureAwait(false))
+            {
+                var nextTargetSubDir = await target.CreateFolderAsync(sourceDirectory.Name, CreationCollisionOption.OpenIfExists).ConfigureAwait(false);
+                await MergeFoldersAsync(sourceDirectory, nextTargetSubDir).ConfigureAwait(false);
+            }
+        }
+        internal async static Task ClearReactDevBundleCacheAsync()
+        {
+            try
+            {
+
+                var devBundleCacheFile = await FileSystem.Current.LocalStorage.GetFileAsync(CodePushConstants.ReactDevBundleCacheFileName).ConfigureAwait(false);
+
+                if (devBundleCacheFile != null)
+                {
+                    await devBundleCacheFile.DeleteAsync().ConfigureAwait(false);
+                }
+            }
+            catch (FileNotFoundException)
```

Comment:
Unfortunately there is no _Try....._ functions. Replaced with preliminary verification if file exists. If file doesn't exists (which is most common case) it simply return no any exception will be thrown.